homework-3/src/test/java/RandomString.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomString {
+    public static void main(String[] args) {
+        String[] stringsList = {"Hello World", "Have a great day", "Angel", "Random String", "Word Game"};
+        List<String> filteredStringList = new ArrayList<String>();
+
+
+        for(Integer i = 0; i < stringsList.length; i++) {
```

Comment:
prefer to use primitive type int instead of Integer object for looping through 'for'